dot/rpc/modules/state_test.go

```diff
@@ -1129,3 +1129,147 @@ func TestStateModuleQueryStorage(t *testing.T) {
 		})
 	}
 }
+
+func TestStateModuleQueryStorageAt(t *testing.T) {
+	t.Parallel()
+	errTest := errors.New("test error")
+
+	type fields struct {
+		storageAPIBuilder func(ctrl *gomock.Controller) StorageAPI
+		blockAPIBuilder   func(ctrl *gomock.Controller) BlockAPI
+	}
+	type args struct {
+		in0 *http.Request
+		req *StateStorageQueryAtRequest
+	}
+	tests := map[string]struct {
+		fields    fields
+		args      args
+		errRegexp string
+		exp       []StorageChangeSetResponse
+	}{
+		"missing_start_block": {
+			fields: fields{
+				storageAPIBuilder: func(ctrl *gomock.Controller) StorageAPI {
+					mockStorageAPI := NewMockStorageAPI(ctrl)
+					mockStorageAPI.EXPECT().GetStorageByBlockHash(&common.Hash{2}, []byte{1, 2, 3}).Return([]byte{1, 1, 1}, nil)
```

Comment:
nit split since it's a bit of a long line 😉 

```suggestion
					mockStorageAPI.EXPECT().GetStorageByBlockHash(&common.Hash{2}, []byte{1, 2, 3}).
						Return([]byte{1, 1, 1}, nil)
```